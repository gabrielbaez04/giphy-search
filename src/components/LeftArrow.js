import React from 'react';
import PropTypes from 'prop-types';

const LeftArrow = ({ goToPrevSlide }) => {
    return (
        <div className="backArrow arrow" onClick={goToPrevSlide}>
            <i className="fa fa-arrow-left fa-2x" aria-hidden="true"></i>
        </div>
    );
}

LeftArrow.propTypes = {
    goToPrevSlide: PropTypes.func.isRequired
}

export default LeftArrow;