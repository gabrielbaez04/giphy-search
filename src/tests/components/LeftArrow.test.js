import { shallow } from 'enzyme';
import LeftArrow from '../../components/LeftArrow';
import React from 'react';

test('should render LeftArrow correctly', () => {
    const wrapper = shallow(<LeftArrow goToPrevSlide={jest.fn()} />);
    expect(wrapper).toMatchSnapshot();
});
