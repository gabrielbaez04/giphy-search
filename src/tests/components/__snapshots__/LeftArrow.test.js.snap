// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render LeftArrow correctly 1`] = `
<div
  className="backArrow arrow"
  onClick={[MockFunction]}
>
  <i
    aria-hidden="true"
    className="fa fa-arrow-left fa-2x"
  />
</div>
`;
